indico/htdocs/js/utils/forms.js

```diff
@@ -100,7 +100,7 @@
 
     global.toggleAclField = function toogleAclField(aclField, state) {
         aclField.closest('.form-group')
-                .find('input.i-button').prop('disabled', state).end()
+                .find('input.i-button, a.i-button').prop('disabled', state).toggleClass('disabled', state).end()
```

Comment:
I'd split it in two lines. the prop makes no sens for a link and the class doesn't make too much sense for a button